editor/src/messages/portfolio/document/node_graph/document_node_definitions.rs

```diff
@@ -37,23 +37,20 @@ pub struct NodePropertiesContext<'a> {
 
 impl NodePropertiesContext<'_> {
 	pub fn call_widget_override(&mut self, node_id: &NodeId, index: usize) -> Option<Vec<LayoutGroup>> {
-		let input_properties_row = self.network_interface.input_properties_row(node_id, index, self.selection_network_path)?;
+		let input_properties_row = self.network_interface.persistent_input_metadata(node_id, index, self.selection_network_path)?;
 		if let Some(widget_override) = &input_properties_row.widget_override {
 			let Some(widget_override_lambda) = INPUT_OVERRIDES.get(widget_override) else {
 				log::error!("Could not get widget override '{widget_override}' lambda in call_widget_override");
 				return None;
 			};
 			widget_override_lambda(*node_id, index, self)
-				.map(|layout_group| {
-					let Some(input_properties_row) = self.network_interface.input_properties_row(node_id, index, self.selection_network_path) else {
-						log::error!("Could not get input properties row in call_widget_override");
-						return Vec::new();
-					};
-					match &input_properties_row.input_data.get("tooltip").and_then(|tooltip| tooltip.as_str()) {
-						Some(tooltip) => layout_group.into_iter().map(|widget| widget.with_tooltip(*tooltip)).collect::<Vec<_>>(),
-						_ => layout_group,
-					}
-				})
+				// .map(|layout_group| {
+				// 	let Some(input_metadata) = self.network_interface.persistent_input_metadata(node_id, index, self.selection_network_path) else {
+				// 		log::error!("Could not get input properties row in call_widget_override");
+				// 		return Vec::new();
+				// 	};
+				// 	layout_group.into_iter().map(|widget| widget.with_tooltip(*input_metadata.input_description.clone())).collect::<Vec<_>>()
+				// })
```

Comment:
Its not necessary to add the tooltip here, since it is handled by the widget in `start_widgets`